mlflow/tracking/client.py

```diff
@@ -274,6 +275,19 @@ def restore_run(self, run_id):
         """
         self.store.restore_run(run_id)
 
+    def search_runs(self, experiment_ids, filter_string, run_view_type):
```

Comment:
On a completely unrelated note, would it make sense for `run_view_type` to have a default value of `ViewType.ACTIVE_ONLY`, since that's what users will be interested in in the majority of cases?